doc/api/repl.md

```diff
@@ -2,80 +2,129 @@
 
     Stability: 2 - Stable
 
-A Read-Eval-Print-Loop (REPL) is available both as a standalone program and
-easily includable in other programs. The REPL provides a way to interactively
-run JavaScript and see the results.  It can be used for debugging, testing, or
-just trying things out.
+The `repl` module provides a Read-Eval-Print-Loop (REPL) implementation that
+is available both as a standalone program or includable in other applications.
+It can be accessed using:
 
-By executing `node` without any arguments from the command-line you will be
-dropped into the REPL. It has simplistic emacs line-editing.
+```js
+const repl = require('repl');
+```
+
+## Design and Features
+
+The `repl` module exports the `repl.REPLServer` class. While running, instances
+of `repl.REPLServer` will accept individual lines of user input, evaluate those
+according to a user-defined evaluation function, then output the result. Input
+and output may be from `stdin` and `stdout`, respectively, or may be connected
+to any Node.js [stream][].
+
+Instances of `repl.REPLServer` support persistent history of prior inputs,
+automatic completion of inputs, simplistic Emacs-style line editing, multi-line
+inputs, ANSI-styled output, saving and restoring current REPL session state,
+error recovery, and customizable evaluation functions.
+
+### Commands and Special Keys
+
+The following special commands are supported by all REPL instances:
+
+* `.break` - When in the process of inputting a multi-line expression, entering
+  the `.break` command (or pressing the `<Ctrl>-C` key combination) will abort
+  further input or processing of that expression.
+* `.clear` - Resets the REPL `context` to an empty object and clears any
+  multi-line expression currently being input.
+* `.exit` - Close the I/O stream, causing the REPL to exit.
+* `.help` - Show this list of special commands.
+* `.save` - Save the current REPL session to a file:
+  `>.save ./file/to/save.js`
```

Comment:
I know it’s only copied, but there should be a space between `>` and `.save`, right?